_posts/patterns/2015-04-15-icons.md

```diff
@@ -877,7 +899,7 @@ info:               Icons are used to clearly and visually represent some inform
             </span>
         </div>
         <div class="icon-classname">
-            Quote
+            quote-left
```

Comment:
Not that it's necessary, but it seems odd that there isn't an analogous `quote-right` in the icon set.